lxd/network/driver_bridge.go

```diff
@@ -273,20 +281,23 @@ func (n *bridge) Validate(config map[string]string) error {
 		// ---
 		//  type: string
 		//  shortdesc: MAC address for the bridge
+		//  scope: global
```

Comment:
I don't think MAC addresses can be global.

Reply:
Here's my thinking: since all cluster members need to have an identical defined network, the bridge MAC address shouldn't differ on a per-member basis.

Reply:
The bridge MAC address outta be unique per member for proper function. Otherwise you get conflicting MAC addresses in your L2 network.

Reply:
If it cant be set per-member today then its global currently.

Whether that is appropriate or not is the matter of a separate GH issue :)

Reply:
A quick test: 

```bash
❯ lxc network create foo --type=bridge --target=devbox bridge.hwaddr=00:00:00:00:00:00
Error: Config key "bridge.hwaddr" may not be used as member-specific key
```

Reply:
I've opened https://github.com/canonical/lxd/issues/14548 to track.